projects/02-dataverse/README.md

```diff
@@ -336,6 +336,7 @@ npm run test:oas // Esto es para correr todos los tests de OAs
   - [ ] Tiene un `<footer>`
   - [ ] Tiene un `<main>` con `<h2>`
   - [ ] Todas las etiquetas de controles (inputs, selects, radio, etc) tienen `<label>`
+  - [ ] Todas las etiquetas `<label>` usan el atributo `for` y el valor de este coincide con el `id` de la etiqueta de control (inputs, selects, radio, etc)
```

Comment:
Acá tengo una duda, ¿hay que hacer un test para cada punto que aparece en [_Criterios de código_](https://github.com/Laboratoria/curriculum/blob/main/projects/02-dataverse/README.md#6-criterios-de-aceptaci%C3%B3n-m%C3%ADnimos-del-proyecto)?
Si es así, este punto lo agrego por que el test que existe de `label `con `input` falta ser un poco más explícito ya que no solo se verifica si todos los `input` tienen `label`, sino que también verifica si el atributo `for` y el valor de este coincide con el `id` de la etiqueta de control.

Reply:
```suggestion
  - [ ] Todas las etiquetas `<label>` usan el atributo `for`
```